src/components/AmountInput/AmountInput.js

```diff
@@ -158,30 +154,29 @@ const watch = {
   inputValue() {
     this.fitTextContainer();
     if (!this.isFocused) { return; }
-    setValue(this.$refs.input, parse(this.numericInputValue));
+    setValue(this.$refs.input, parse(this.numericInputValue, { ...this.currencyOptions, currency: this.currency }));
```

Comment:
where this `setValue` came from? some times I see it with 1 argument and some others like this one, with 2)... could you explain me a bit? Thanks!!!!

Reply:
`setValue` and `parse` are helpers from `vue-currency-input`. `setValue` only accepts 2 arguments: the HtmlElement and the value. `parse`, on the other hand, accepts one or two arguments: value and an object with options.
Maybe I would separate the value set into it's own line, for better understading

Reply:
Thanks!